pkg/config/config.go

```diff
@@ -14,35 +15,56 @@ const (
 )
 
 type Config struct {
-	Verbose               bool
-	WebappPort            string
-	ClairAddress          string
-	KlarTrace             bool
-	KlarResultServicePath string
-	KlarResultListenPort  string
+	Verbose                  bool
+	WebappPort               string
+	ClairAddress             string
+	KlarTrace                bool
+	KlarResultServicePath    string
+	KlarResultListenPort     string
+	KlarResultServiceAddress string
 }
 
 func setConfigDefaults() {
 	viper.SetDefault(Verbose, "false")
 	viper.SetDefault(ListeningPort, "8080")
 	viper.SetDefault(KlarResultServicePath, "http://kubei.kubei:8081/result/")
 	viper.SetDefault(KlarResultListenPort, "8081")
-	viper.SetDefault(KlarTrace, "false")                   // Run Klar in more verbose mode
+	viper.SetDefault(KlarTrace, "false") // Run Klar in more verbose mode
 	viper.SetDefault(ClairAddress, "clair.kubei")
 
 	viper.AutomaticEnv()
 }
 
+// extracts the service address from the full path.
+// example: `http://kubei.kubei:8081/result/` should return `kubei.kubei`
+func getServiceAddress(serviceFullPath string) string {
```

Comment:
This can be done using url.Parse("http://kubei.kubei:8081/result/"):
	u, err := url.Parse("http://kubei.kubei:8081/result/")
	if err != nil {
		....
	}
	fmt.Println(u.Hostname())

Reply:
Thanks, from url.Parse() comments	
```
Trying to parse a hostname and path without a scheme is invalid
 but may not necessarily return an error, due to parsing ambiguities.
```

I will add a validation that the path includes scheme and use url.Parse()